src/initial_conditions/initial_conditions.jl

```diff
@@ -358,6 +358,101 @@ function (initial_condition::RisingThermalBubbleProfile)(params)
     return local_state
 end
 
+"""
+    RCEMIPIIProfile(; temperature = 300)
+
+An `InitialCondition` following the sounding to initialize simulations for
+RCEMIPII as described by Wing et. al. (2018). Note - this should be used
+for RCE_small and NOT RCE_large - RCE_large must be initialized with the
```

Comment:
Would be good to add some checks to `check_case_consistency` to validate that relevant flags are applied / not applied.